.env

```diff
@@ -109,6 +109,6 @@ GRAFANA_IMAGE_TAG=8.5.16
 
 ### Block Node ####
 BLOCK_NODE_IMAGE_PREFIX=ghcr.io/hiero-ledger/
-BLOCK_NODE_IMAGE_TAG=0.9.0
+BLOCK_NODE_IMAGE_TAG=0.10.0-rc3
```

Comment:
```suggestion
BLOCK_NODE_IMAGE_TAG=0.10.0
```

Reply:
Done. Thanks. 👍

Reply:
However 0.10.0 is still not released in github packages, therefore tests are failing. We need to postpone this PR until it's released